src/main/java/org/economicsl/auctions/singleunit/Service.java

```diff
@@ -0,0 +1,21 @@
+// Copyright (c) 2017 Robert Bosch GmbH
```

Comment:
@davidrpugh This file is just for testing purposes. We need to get rid of it before merging the stuff back.